specs/CriticalUpdateAvailable.md

```diff
@@ -0,0 +1,106 @@
+Critical Update Available
+===
+
+# Background
+WebView2 developers often have to author ECS configurations to toggle feature flags. However, once 
```

Comment:
Is 'WebView2 developers' us? If so it might be better to say. Also will be nice to define ECS.

```suggestion
As WebView2 developers, we often have to author ECS (experimentation and configuration service - used to perform A/B testing or remotely disable features) configurations to toggle feature flags. However, once 
```